common/beerocks/bwl/dummy/ap_wlan_hal_dummy.cpp

```diff
@@ -376,9 +376,12 @@ bool ap_wlan_hal_dummy::process_dummy_event(parsed_obj_map_t &parsed_obj)
             LOG(ERROR) << "Failed reading mac parameter!";
             return false;
         }
+        msg->params.association_frame = new char[30];
+        msg->params.mac               = beerocks::net::network_utils::mac_from_string(tmp_str);
+        char assoc_req[]              = "dummy association frame";
```

Comment:
Not sure why it is required to write `"dummy association frame"` on the tlv, but anyway it probably will better if you use std::string instead of c-string.